src/main/java/com/hubspot/jinjava/lib/filter/Base64DecodeFilter.java

```diff
@@ -0,0 +1,52 @@
+package com.hubspot.jinjava.lib.filter;
+
+import com.hubspot.jinjava.doc.annotations.JinjavaDoc;
+import com.hubspot.jinjava.doc.annotations.JinjavaParam;
+import com.hubspot.jinjava.doc.annotations.JinjavaSnippet;
+import com.hubspot.jinjava.interpret.InvalidArgumentException;
+import com.hubspot.jinjava.interpret.InvalidReason;
+import com.hubspot.jinjava.interpret.JinjavaInterpreter;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+@JinjavaDoc(
+  value = "Decode a base 64 input into a string.",
+  input = @JinjavaParam(
+    value = "input",
+    type = "string",
+    desc = "The base 64 input to decode.",
+    required = true
+  ),
+  params = {
+    @JinjavaParam(
+      value = "encoding",
+      type = "string",
+      desc = "The string encoding charset to use.",
+      defaultValue = "UTF-8"
+    )
+  },
+  snippets = {
+    @JinjavaSnippet(code = "{% set my_number = -53 %}\n" + "{{ my_number|abs }}")
```

Comment:
I think you copied this over from `AbsFilter` - should probably update it to use `b64decode`

Reply:
Indeed, thanks for catching this!